Libraries/LibWeb/HTML/HTMLDialogElement.cpp

```diff
@@ -277,19 +278,24 @@ WebIDL::ExceptionOr<void> HTMLDialogElement::show_a_modal_dialog(HTMLDialogEleme
 void HTMLDialogElement::close(Optional<String> return_value)
 {
     // 1. If returnValue is not given, then set it to null.
-    // 2. Close the dialog this with returnValue.
-    close_the_dialog(move(return_value));
+    // 2. Close the dialog this with returnValue and null.
+    close_the_dialog(move(return_value), nullptr);
 }
 
 // https://html.spec.whatwg.org/multipage/interactive-elements.html#dom-dialog-requestclose
 void HTMLDialogElement::request_close(Optional<String> return_value)
 {
+    // ADHOC: This method is an amalgamation of the requestClose() and "request to close" algorithms from the spec.
+    // FIXME: Once the "request-close" command is implemented, this will need to be split into two methods.
+    // For now this implementation is only used for the requestClose() method, which sets source to null.
```

Comment:
We also hang fixme/ad-hoc indents if it flows into multiple lines:
```c++
// FIXME: Once the "request-close" command is implemented, this will need to be split into two methods.
//        For now this implementation is only used for the requestClose() method, which sets source to null.
```